aws-ts-static-website/index.ts

```diff
@@ -26,16 +26,13 @@ const config = {
 };
 
 // contentBucket is the S3 bucket that the website's contents will be stored in.
-const contentBucket = new aws.s3.Bucket("contentBucket",
-    {
-        bucket: config.targetDomain,
-        // Configure S3 to serve bucket contents as a website. This way S3 will automatically convert
-        // requests for "foo/" to "foo/index.html".
-        website: {
-            indexDocument: "index.html",
-            errorDocument: "404.html",
-        },
-    });
+const contentBucket = new aws.s3.BucketV2("contentBucket", {bucket: config.targetDomain});
```

Comment:
Don't we have to make this bucket publicly readable?

Reply:
I think that's right. The example doesn't provision otherwise. Checking.

Reply:
I don't think we do since we are configuring public access through a CloudFront distribution with accesses the bucket via an OIC and bucket policies.

Reply:
Your intuition is right. I mixed up some test results. It works without it, therefore I will remove.

Reply:
I think sadly ACL is still needed for the logs bucket. Just not the content bucket.

Reply:
Let me update Python to match.